docs/edge.md

```diff
@@ -108,19 +106,24 @@ How often should Edge run in seconds? <br/>
 
 #### calculate_since_number_of_days
 Number of days of data against which Edge calculates Win Rate, Risk Reward and Expectancy
-Note that it downloads historical data so increasing this number would lead to slowing down the bot<br/>
+Note that it downloads historical data so increasing this number would lead to slowing down the bot.<br/>
 (default to 7)
 
+#### capital_available_percentage
+This is the percentage of the total capital on exchange in stake currency. <br/>
+As an example if you have 100 USDT available in your wallet on the exchange and this value is 0.5 (which is 50%), then the bot will use a maximum amount of 50 USDT for trading and considers it as available capital.<br/>
+(default to 0.5)
```

Comment:
when is this calculated? 
on startup or every other iteration?

assuming i got out of a trade (or put more money into the exchange) and now have 200USDT available - 2 days after running the bot?

And honestly, please stick to 1 stake-currency within this document.... 20 lines higher ETH is your stake, now it's USDT... makes following the logic slightly confusing - and it doesn't really matter which is used as sample.

Reply:
I will enrich this part. I am so into it that I can hardly put myself in shoes of a new reader.

Reply:
that's why we do reviews